defaults/main.yml

```diff
@@ -6,21 +6,20 @@ odoo_role_odoo_group: odoo
 # VirtualEnv vars
 odoo_role_odoo_venv_path: /opt/.odoo_venv
 
-odoo_role_odoo_edition: oca
-
 # Edition vars
+
 # Vars for the Odoo Nightly edition
-# odoo_role_odoo_edition: "odoo"
+odoo_role_download_strategy: tar
 odoo_role_odoo_version: 11.0
-odoo_role_odoo_release: 20170914
+odoo_role_odoo_release: 20190501
 odoo_role_odoo_url: "https://nightly.odoo.com/{{ odoo_role_odoo_version }}/nightly/src/odoo_{{ odoo_role_odoo_version }}.{{ odoo_role_odoo_release }}.tar.gz"
-odoo_role_odoo_download_path: "/tmp/odoo_{{ odoo_role_odoo_version }}.{{ odoo_role_odoo_release }}.tar.gz"
+odoo_role_odoo_download_path: "{{ odoo_role_odoo_path }}/../odoo_releases/odoo_{{ odoo_role_odoo_version }}.{{ odoo_role_odoo_release }}.tar.gz"
```

Comment:
"odoo_releases" implies that in this directory we could end up with multiple files, right? Should we keep all the old releases?

Reply:
yes, but I didn't go further. I can think on a way to remove its contents if inside there isn't the package we want

Reply:
we can leave it like that for now for this first iteration IMO